src/App.js

```diff
@@ -1,29 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 
```

Comment:
- [ ] Please, consider removing all the boilerplate that got created by the CRA (texts, images, and styles) - just clear your folder structure 👍 

![image](https://user-images.githubusercontent.com/49064106/201467336-06a3ca07-63d8-4386-ae6e-3229ed8eb640.png)